src/aggregate.py

```diff
@@ -0,0 +1,87 @@
+import csv
+import datetime
+import os
+
+import constants
+from helpers import data
+from helpers import merging
+
+
+def load_data_dir(dir_name):
```

Comment:
Would you mind adding brief docstrings to these functions?

Reply:
Done